packages/uswds-core/src/styles/mixins/general/table.scss

```
@@ -144,6 +144,10 @@ $table-unsorted-icon-color: get-color-token-from-bg(
     display: none;
   }
 
+  th {
+    background: color($table-background-color);
```

Is it important that this is `background` not `background-color`?

---

Good catch. It makes sense to me to switch it to background-color here. Initial testing showed no regressions and it seems like there are no conflicting `background` styles that this is trying to override. 
```suggestion
    background-color: color($table-background-color);
```